src/Analysis/GWASApp/Components/Covariates/ContinuousCovariates.jsx

```diff
@@ -25,7 +25,11 @@ const ContinuousCovariates = ({
     <React.Fragment>
       <div className='GWASUI-flexRow continuous-covariates'>
         <div className='continous-covariates-table'>
-          <Covariates selected={selected} handleSelect={setSelected} />
+          <Covariates
+            selected={selected}
+            handleSelect={setSelected}
+            selectedCovariates={outcome?.concept_id ? [outcome.concept_id, ...selectedCovariates.map((obj) => obj.concept_id)] : []}
```

Comment:
would it be possible to use the already existing `selected` instead? If not, can you please add a comment explaining why not? And perhaps consider another name so `selectedCovariates` and `selected` are not easily mixed up?

Reply:
the selected is for what the user currently has selected in the list, my new variable is for what they already have selected and submitted i will try to think of a new better name for this. Probably `submittedCovariates` if that works for you

Reply:
Ok, `submittedCovariates` sounds good to me.